lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
It is clear that the api is not changed by this PR.
However I do think that when new code that is added - and this PR does add a new method for the client users, it can and should be done better than the past if possible. Why add something if you already know you'd like to change it after? Otherwise the client will be always compliant with the past and it will be really hard to improve it. It's not a critical issue, just my 2 cents.